basic/x_series_actuator/01a_lookup.cpp

```diff
@@ -38,4 +38,6 @@ int main() {
     << "  on the local network.  If this is empty make sure that the modules" << std::endl
     << "  are connected, powered on, and that the status LEDs are displaying" << std::endl
     << "  a green soft-fade." << std::endl;
+
+  return 0;
```

Comment:
Interesting bit in the C++ standard:

> A return statement in `main` has the effect of leaving the main function (destroying any objects with automaticstorage duration) and calling `std::exit` with the return value as the argument. If control flows off the end of the compound-statement of main, the effect is equivalent to a return with operand 0

So with `main`, you technically don't have to even specify a `return` statement

Reply:
huh -- interesting.  That makes me feel better about how it was!

It is weird, though -- I could have sworn I got a warning that triggered me adding this.  :man_shrugging: